scripts/edit-task.js

```diff
@@ -0,0 +1,19 @@
+import { CLICK_EVENT } from "/scripts/constants.js";
+import onEditButtonClick from "/scripts//events/onEditButtonClick.js";
+
+function onTaskEdit(buttons, inputElement, toolbar, tasks, id) {
+  function editEventHandler() {
+    onEditButtonClick(
+      buttons,
+      inputElement,
+      toolbar,
+      tasks,
+      id,
+      editEventHandler
+    );
+  }
+  const editButton = buttons[2];
```

Comment:
unused variable.

Reply:
@sam-wtag editButton is used in the 16th line.
`editButton.addEventListener(CLICK_EVENT, editEventHandler);`